sway/sway-output.5.scd

```diff
@@ -190,6 +190,19 @@ must be separated by one space. For example:
 	may have no effect or produce unexpected output when used together with future
 	HDR support features.
 
+*output* <name> allow_tearing yes|no
+	Allows or disallows screen tearing as a result of asynchronous page flips,
+	and an immediate presentation mode from a client.
+
+	With asynchronous page flips, frames from the client are presented as soon
+	as possible instead of synchronizing with the monitor's vblank interval 
+	(VSync). This prevents stutter and reduces latency in games.
+
+	To adjust whether tearing is allowed for specific applications, see
+	*allow_tearing* in *sway*(5).
```

Comment:
Maybe we could repeat here that tearing will only be enabled when it's allowed for both the output and the application.

Reply:
Added an extra comment to explain that tearing will only be enabled when it's allowed for the output and application.